logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/backup/ImportBackupUseCase.kt

```diff
@@ -0,0 +1,13 @@
+package com.wire.kalium.logic.feature.backup
+
+import com.wire.kalium.persistence.backup.BackupImporter
+
+// TODO: User IO Dispatcher
+class ImportBackupUseCase(
+    private val backupImporter: BackupImporter
+) {
+
+    suspend operator fun invoke(backupFilePath: String) {
```

Comment:
Maybe add some inline documentation?
/**
     * Imports a valid previously created backup file into the current database, respecting the current data if there is any overlap.
     * @param backupFilePath The absolute file system path to the backup file
     */

Reply:
Could we also make this use case implement an interface so that we can add some unit tests to it? In this context, having this use case return something like **ImportBackupUseCaseResult** would be really helpful so that the call result can be error handled on the VM layer later on.